inc/actions.php

```diff
@@ -291,3 +287,31 @@ function uploadcare_settings() {
 function uploadcare_settings_actions() {
     add_options_page('Uploadcare', 'Uploadcare', 'upload_files', 'uploadcare', 'uploadcare_settings');
 }
+
+/*
+ * Calculate final image dimensions
+ */
+function uploadcare_get_final_dim($file) {
```

Comment:
if this is rare operation (e.g. happens only on file upload), we can just send HEAD to cdn url and get the dimensions in headers or get `/json/`

```
curl -I https://ucarecdn.com/e8ebfe20-8c11-4a94-9b40-52ecad7d8d1a/-/preview/100x100/
HTTP/2 200 
access-control-allow-methods: HEAD, GET, OPTIONS
...
x-image-height: 100
x-image-width: 100
...

curl https://ucarecdn.com/e8ebfe20-8c11-4a94-9b40-52ecad7d8d1a/-/preview/100x100/-/json/ 
{ "width": 100,
  "height": 100,
  "id": "/e8ebfe20-8c11-4a94-9b40-52ecad7d8d1a/-/preview/100x100/",
  ...
}%
```

this is future proof as we don't have to think here about any operations that may be added to API that we have to add to switch/if statements.